### ISSUE TITLE
Meeting participants hold all the same person

### ISSUE THREAD

**Issue by yuya-tani** [trogers-twilio/plugin-external-conference-warm-transfer#4]
Hello.

I often use this library. Thank you.

The question is, if you have more than two people in a conference call,
If you put the third person on hold, the second person will be on hold.
Where should I fix it?

Best Regard.

**Comment by jessycormier** [comment]
I believe you need to change what SID is being used to put calls on hold, ATM it's most likely set to use the conference SID which puts the everyone in that state, try digging around with the caller sids and see if you can then trigger the proper mute.

**Comment by oleksiyboyko** [comment]
@yuya-tani 
Hello.
Did you manage to solve this problem?
Best Regard

**Comment by yuya-tani** [comment]
I'm very very sorry.

I asked this question and got an answer, but I didn't reply.
@jessycormier
Because it was the SID of Conference as you answered,
I should have looked into Caller.

In the end, the specifications on the customer side changed, so I responded with another method instead of this library.
I forgot to report that point.

I'm sorry that I didn't reply even though I received an answer.
It's okay to hate me.

@alekseyboyko 
Thanks for reminding me that I forgot to reply.
I am very sorry for what I did.

Thank you both.
